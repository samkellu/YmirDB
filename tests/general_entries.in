set c 7
set b 3 5 c
set a 1 2 b
del a
snapshot
bye
