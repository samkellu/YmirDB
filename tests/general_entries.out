> ok

> ok

> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> b [1 2 3]
c [7]

> not permitted

> ok

> bye
